templates/icinga.ini

```diff
@@ -298,5 +298,276 @@ min = "0"
+items.value = "$service_name_template$.perfdata.api_num_json_rpc_sync_queue_items.value"
+item_rate.value = "$service_name_template$.perfdata.api_num_json_rpc_sync_queue_item_rate.value"
+
+[icinga-api-num-json-rpc-sync-queue.urlparams]
+title = "API JSON RPC Sync Queue"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-api-num-json-rpc-sync-queue.functions]
+items.value = "alias(color($metric$, '#1a7dd7'), 'Items')"
+item_rate.value = "alias(color($metric$, '#0b3c68'), 'Items/s')"
+
+
+[icinga-idomysqlconnection-ido-mysql-queries.graph]
+check_command = "icinga"
+
+[icinga-idomysqlconnection-ido-mysql-queries.metrics_filters]
+1min.value = "$service_name_template$.perfdata.idomysqlconnection_ido-mysql_queries_1min.value"
+5mins.value = "$service_name_template$.perfdata.idomysqlconnection_ido-mysql_queries_5mins.value"
+15mins.value = "$service_name_template$.perfdata.idomysqlconnection_ido-mysql_queries_15mins.value"
+
+[icinga-idomysqlconnection-ido-mysql-queries.urlparams]
+title = "IDO MySQL Connection Queries"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-idomysqlconnection-ido-mysql-queries.functions]
+1min.value = "alias(color($metric$, '#61aaed'), 'Queries/1m')"
+5mins.value = "alias(color($metric$, '#1a7dd7'), 'Queries/5m')"
+15mins.value = "alias(color($metric$, '#0b3c68'), 'Queries/15m')"
+
+
+[icinga-passive-host-checks.graph]
+check_command = "icinga"
+
+[icinga-passive-host-checks.metrics_filters]
+1min.value = "$service_name_template$.perfdata.passive_host_checks_1min.value"
+5min.value = "$service_name_template$.perfdata.passive_host_checks_5min.value"
+15min.value = "$service_name_template$.perfdata.passive_host_checks_15min.value"
+
+[icinga-passive-host-checks.urlparams]
+title = "Passive Host Checks"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-passive-host-checks.functions]
+1min.value = "alias(color($metric$, '#61aaed'), 'Checks/1m')"
+5min.value = "alias(color($metric$, '#1a7dd7'), 'Checks/5m')"
+15min.value = "alias(color($metric$, '#0b3c68'), 'Checks/15m')"
+
+
+[icinga-api-num-json-rpc-anonymous-clients.graph]
+check_command = "icinga"
+
+[icinga-api-num-json-rpc-anonymous-clients.metrics_filters]
+clients.value = "$service_name_template$.perfdata.api_num_json_rpc_anonymous_clients.value"
+
+[icinga-api-num-json-rpc-anonymous-clients.urlparams]
+title = "API JSON RPC Anonymous Clients"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-api-num-json-rpc-anonymous-clients.functions]
+clients.value = "alias(color($metric$, '#61aaed'), 'Clients')"
+
+
+[icinga-current-concurrent-checks.graph]
+check_command = "icinga"
+
+[icinga-current-concurrent-checks.metrics_filters]
+checks.value = "$service_name_template$.perfdata.current_concurrent_checks.value"
+
+[icinga-current-concurrent-checks.urlparams] 
+title = "Current Concurrent Checks"
+areaAlpha = "0.5"
+areaMode = "all"
```

Comment:
It seems your graphs with exactly one metric sometimes contain a transparent area and sometimes no area at all. Please unify.